docs/users.md

```diff
@@ -5,6 +5,7 @@ Users represent an individual's account on Box.
 
 * [Get the Current User's Information](#get-the-current-users-information)
 * [Update User](#update-user)
+* [Invite User](#invite-user)
```

Comment:
We should clarify this as "Invite User to Enterprise", to avoid confusion with inviting a user to collaborate on an item.